### ISSUE TITLE
open outside website in new tab

### ISSUE THREAD

**Issue by pupboss** [Kikobeats/uno-zen#20]
add the follow code into uno.js

```
$(document.links).filter(function() {
    return this.hostname != window.location.hostname;
}).attr('target', '_blank');
```

and the links in post outside site would be opened in a new tab

**Comment by Kikobeats** [comment]
hmm good point, I saw that you copied the code of [this post](https://ghost.org/forum/using-ghost/1167-link-targets/) in Ghost forum. The only thing that makes me feel unsafe is the perfomance about this. Keep in mind that that the theme is designed for tablet/mobile too and do the action in a mobile can be a little overload.

what do you think?

**Comment by pupboss** [comment]
Yeah, you are right.

I forgot some people use mobile more often = =

**Comment by Kikobeats** [comment]
@pupboss I'm checking that for desktop version can be very useful, because you can manage more tabs, so we can put the code and only load the code under desktop or table detection

**Comment by pupboss** [comment]
What's up

**Comment by pupboss** [comment]
That would be great!
Sometimes owner's article contain some material, if we don't do that, readers would lose the point, they have to return to the earlier site
So if you add this on desktop version UE would be better

**Comment by Kikobeats** [comment]
available in the next `2.2.0` tagged version :-)
